blamo/lib/Unit/units.dart

```diff
@@ -269,7 +268,7 @@ Widget getScaffold(List<Unit> units){
     }
   }
 
-void testBuiildingList() async {
+  void testBuiildingList() async {
```

Comment:
Is this function being used for anything? Can probably be removed or at least typo fixed in name.